test/kokoro/xds_k8s_lb.sh

```diff
@@ -167,8 +167,10 @@ main() {
     "outlier_detection_test"
     "remove_neg_test"
     "round_robin_test"
-    "bootstrap_generator_test"
   )
+  if [ "${TESTING_VERSION}" != "master" ]; then
```

Comment:
Prefer `[` over `[[`: https://google.github.io/styleguide/shellguide.html#s6.3-tests.

Reply:
Ah, interesting, changed. Noted for future.